package.json

```diff
@@ -41,7 +41,7 @@
     "codeclimate-test-reporter": "^0.4.0",
     "colors": "^1.1.2",
     "css-loader": "^0.26.1",
-    "d2": "^26.1.7",
+    "d2": "^26.1.12",
```

Comment:
But they lock to 26.1.9 https://github.com/dhis2/d2-ui/blob/v26/yarn.lock#L1563. But regarless what the lock says, in package.json we should probably point to 26.1.7, otherwise it's indeed confusing.